core-bundle/contao/templates/forms/form_altcha.html5

```diff
@@ -0,0 +1,17 @@
+<?php $this->extend('form_row'); ?>
+
+<?php $this->block('label'); ?>
+  <?php if ($this->label): ?>
+    <label for="ctrl_<?= $this->id ?>"<?php if ($this->class): ?> class="<?= $this->class ?>"<?php endif; ?>>
+      <span class="invisible"><?= $this->mandatoryField ?> </span><?= $this->label ?><span class="mandatory">*</span>
+    </label>
+  <?php endif; ?>
+<?php $this->endblock(); ?>
+
+<?php $this->block('field'); ?>
+    <?php if ($this->hasErrors()): ?>
+      <p class="error"><?= $this->getErrorAsString() ?></p>
+    <?php endif; ?>
+
+    <altcha-widget <?= $this->strAltchaAttributes ?>></altcha-widget>
```

Comment:
Here, you could just place the `HtmlAttributes` instance which would make it a lot simpler to add something :)

Reply:
Yes, that‘s way better. Now we can simply do `$this->altchaAttributes->set('foo', 'bar')`. 👍